packages/json-csv-core/tsconfig.json

```diff
@@ -3,8 +3,6 @@
 	"extends": "../../tsconfig-base.json",
 	"compilerOptions": {
 		"allowSyntheticDefaultImports": true,
-		"inlineSourceMap": true,
```

Comment:
Disabled by default; source/declaration maps are toggled off in tsconfig.build.json